text/0097-log-data-model.md

```diff
@@ -1002,6 +1003,326 @@ span_id          | string             | The span ID within the trace associated
+    <td>resource[“net.peer.ip”] or resource[“net.host.ip”]</td>
+  </tr>
+  <tr>
+    <td>cloud.account.id</td>
+    <td>string</td>
+    <td>ID of the account in the given cloud</td>
+    <td>resource[“cloud.account.id”]</td>
+  </tr>
+  <tr>
+    <td>cloud.availability_zone</td>
+    <td>string</td>
+    <td>Availability zone in which this host is running.</td>
+    <td>resource[“cloud.zone”]</td>
+  </tr>
+  <tr>
+    <td>cloud.instance.id</td>
+    <td>string</td>
+    <td>Instance ID of the host machine.</td>
+    <td>**resource</td>
+  </tr>
+  <tr>
+    <td>cloud.instance.name</td>
+    <td>string</td>
+    <td>Instance name of the host machine.</td>
+    <td>**resource</td>
+  </tr>
+  <tr>
+    <td>cloud.machine.type</td>
+    <td>string</td>
+    <td>Machine type of the host machine.</td>
+    <td>**resource</td>
+  </tr>
+  <tr>
+    <td>cloud.provider</td>
+    <td>string</td>
+    <td>Name of the cloud provider. Example values are aws, azure, gcp, or digitalocean.</td>
+    <td>resource[“cloud.provider”]</td>
+  </tr>
+  <tr>
+    <td>cloud.region</td>
+    <td>string</td>
+    <td>Region in which this host is running.</td>
+    <td>resource[“cloud.region”]</td>
+  </tr>
+  <tr>
+    <td>cloud.image.id*</td>
+    <td>string</td>
+    <td></td>
+    <td>resource[“host.image.name”]</td>
+  </tr>
+  <tr>
+    <td>container.id</td>
+    <td>string</td>
+    <td>Unique container id</td>
+    <td>resource[“container.id”]</td>
+  </tr>
+  <tr>
+    <td>container.image.name</td>
+    <td>string</td>
+    <td>Name of the image the container was built on.</td>
+    <td>resource[“container.image.name”]</td>
+  </tr>
+  <tr>
+    <td>container.image.tag</td>
+    <td>Array of string</td>
+    <td>Container image tags.</td>
+    <td>**resource</td>
+  </tr>
+  <tr>
+    <td>container.labels</td>
+    <td>key/value</td>
+    <td>Image labels.</td>
+    <td>attributes[*]</td>
+  </tr>
+  <tr>
+    <td>container.name</td>
+    <td>string</td>
+    <td>Container name.</td>
+    <td>resource[“container.name”]</td>
+  </tr>
+  <tr>
+    <td>container.runtime</td>
+    <td>string</td>
+    <td>Runtime managing this container. Example: “docker”</td>
+    <td>**resource</td>
+  </tr>
+  <tr>
+    <td>destination.address</td>
+    <td>string</td>
+    <td>Destination address for the event</td>
+    <td>attributes[“destination.address”]</td>
+  </tr>
+  <tr>
+    <td>error.code</td>
+    <td>string</td>
+    <td>Error code describing the error.</td>
+    <td>attributes[“error.code”]</td>
+  </tr>
+  <tr>
+    <td>error.id</td>
+    <td>string</td>
+    <td>Unique identifier for the error.</td>
+    <td>attributes[“error.id”]</td>
+  </tr>
+  <tr>
+    <td>error.message</td>
+    <td>string</td>
+    <td>Error message.</td>
+    <td>attributes[“error.stack_trace”]</td>
+  </tr>
+  <tr>
+    <td>error.stack_trace</td>
+    <td>string</td>
+    <td>The stack trace of this error in plain text.</td>
+    <td>**resource</td>
```

Comment:
Why is stack trace mapped to `resource`? Resource cannot change per log record, but stack trace can.